src/Profiteur/DataType.hs

```diff
@@ -0,0 +1,8 @@
+module Profiteur.DataType where
+
+import Data.String
+
+data DataType = JQuery | DataFile FilePath deriving (Show)
```

Comment:
Can we call this datatype `DataFile` and use `JQueryFile` for the current `JQuery`?